packages/es-dev-server/TODO.md

```diff
@@ -0,0 +1,21 @@
+Before merge:
+- check if karma still works
+- proxy configuration
+- custom middleware
+- make transforming index.html async
+- await running async task in index.html
+- index html transform === slow, speed it up somehow?
+- error logging
+  - inline module
+  - no external module found
+- custom SSL key / cert
+- ensure no commonjs, amd etc.
+
+Later:
+- make polyfills configurable
+- format load script
+- support typescript somehow
+- look into babel specificity: js config overrides babelrc?
```

Comment:
if you are talking about `babel.config.js` then it's a feature
see https://babeljs.io/docs/en/config-files#project-wide-configuration

Reply:
Not entirely. I'm calling babel from javascript using a config. I'm wondering if the user's own configuration will overwrite that, or if I will overwrite the user's configuration.